lib/Doctrine/Common/Collections/ExpressionBuilder.php

```diff
@@ -52,7 +52,7 @@ public function andX($x = null)
      */
     public function orX($x = null)
     {
-        return new CompositeExpression(CompositeExpression::TYPE_OR, func_get_args());
+        return new CompositeExpression(CompositeExpression::TYPE_OR, is_array($x) ? $x : func_get_args());
```

Comment:
This should be changed to just accept `$x`, and the method signature should become `...$x`